apps/web/src/components/pricing/standardPricingCard.tsx

```diff
@@ -53,7 +53,7 @@ export default function CardStandard({ className, ...props }: CardProps) {
         ))}
       </CardContent>
       <CardFooter>
-        {/* TODO: redirect to shop */}
+        {/* TODO - redirect to shop */}
```

Comment:
Why the reverse?

Reply:
Overlooked while resolving merge conflicts.